packages/compass-e2e-tests/helpers/commands/get-connect-form-state.ts

```diff
@@ -191,6 +191,26 @@ export async function getConnectFormState(
     ),
   });
 
+  // FLE2
```

Comment:
Side note that is totally irrelevant for review: I am not sure what to do with our internal usage of “CSFLE” (as an umbrella term for CSFLE+QE) and “FLE2”. It’s just naming, and it’s all internal, but should we worry about people down the line being confused by our outdated terminology? Is it worth doing a huge renaming of everything?

Reply:
Absolutely agree that we need to unify the name!! For me this is also confusing.

Reply:
Thx! ✨